pkgs/development/misc/jupyterKernels/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib, pkgs, stdenv}:
+
+let
+
+  defaultKernelDefinition = {
+    python3 = let
+      env = (pkgs.python3.withPackages (pythonPackages: with pythonPackages; [
+              ipykernel
+              pandas
+              scikitlearn
+            ]));
+    in {
+      displayName = "Python 3";
+      argv = [
+        "${env.interpreter}"
+        "-m"
+        "ipykernel_launcher"
+        "-f"
+        "{connection_file}"
+      ];
+      language = "python";
+      logo32 = "${env.sitePackages}/ipykernel/resources/logo-32x32.png";
+      logo64 = "${env.sitePackages}/ipykernel/resources/logo-64x64.png";
+    };
+  };
+
+in
+{
+  defaultKernelDefinition = defaultKernelDefinition;
+  create = { kernelDefinitions ?  defaultKernelDefinition }:
```

Comment:
@FRidh I'm not certain this is better.
Now I'm exporting a set instead of exporting a function so I can re-use the default value in all the packages that might use it.